### DISCUSSION TITLE
Historical Thunderstorm data North America

### DISCUSSION THREAD

**Discussion by canegru** [open-meteo/open-meteo#1016] [General]
We've implemented Open Meteo's API and we lack a way to know if thunderstorms were detected in the area. 

Any idea on how we can gather this data? Do we have to pull the events from NOAA directly?

**Comment by patrick-zippenfenig** [comment]
Thunderstorm are not directly forecast by weather models. They can be parameterised from CAPE, Lifted Index, CIN, precipitation and/or precipitation probability. The current parameterisation can be found here: https://github.com/open-meteo/open-meteo/blob/main/Sources/App/Helper/WeatherCode.swift#L38

The thresholds may change in the future. Ideally, I would like to use ensemble models to better approximate the likelihood of thunderstorms.
